### DISCUSSION TITLE
LV2 Plugins

### DISCUSSION THREAD

**Discussion by jonandtice** [monocasual/giada#624] [General]
Does giada support or plan to support LV2 plugins? I've mainly tried to depend on LV2 plugins on my Linux installation.

**Comment by gvnnz** [comment]
Hi @jonandtice, we added experimental LV2 support in [Giada version 0.23](https://github.com/monocasual/giada/releases/tag/v0.23.0), a couple of months ago. You might want to give it a spin and see if your LV2 plug-ins work as expected. Please report back any weirdness. Thanks!
